app/views/decidim/vocdoni/admin/census/_decidim_permissions_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= decidim_form_for form, url: election_census_path(election), method: :post do |form| %>
+  <%= form.hidden_field :census_permissions_form, value: true %>
+  <% current_organization.available_authorizations.each do |authorization| %>
+    <%= form.check_box :verification_types, { label: t("decidim.authorization_handlers.#{authorization}.name"), multiple: true, include_hidden: false, checked: election.verification_types.include?(authorization) }, authorization %>
```

Comment:
By using the same label we're introducing and accessibility bug here. This is visible because you can't click in the label to select a checkbox. Also, this may introduce problems with screen readers (like NVDA). Can you fix that please?